stats/kafka/utils.go

```diff
@@ -0,0 +1,38 @@
+package kafka
+
+import (
+	"encoding/json"
+
+	"github.com/loadimpact/k6/stats"
+	"github.com/loadimpact/k6/stats/influxdb"
+	jsonCollector "github.com/loadimpact/k6/stats/json"
+)
+
+func formatSamples(format string, samples []stats.Sample) ([]string, error) {
```

Comment:
Please add a *very* simple test for this. I doubt we'll have any issues with the influxdb line-protocol changing any time soon, but since this collector and the json and influxdb ones will be now linked through this function, it's possible for some future refactoring of the other collectors to inadvertently break this.